README.md

```diff
@@ -22,12 +25,104 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
 - Distributed training support
 - TensorBoard support
 
+## Support
+
+### [Latest release v0.7.2](https://github.com/awslabs/sagemaker-debugger/releases)
+    
+   - Experimental support for TF 2.x GradientTape - Introducing experimental support for TF 2.x training scripts using GradientTape. With this change, weights, bias, loss, metrics, and gradients are captured by SageMaker Debugger. These changes work with vanilla version of Tensorflow 2.x (not with the zero-code change version).
```

Comment:
you could reword this last sentence to say that these changes will work with changes to training script. it does not work with zero script change experience at this time